make_queue/views/reservation/reservation.py

```diff
@@ -14,6 +15,26 @@
 from make_queue.util.time import timedelta_to_hours
 from news.models import TimePlace
 
+from django.core.mail import send_mail, EmailMessage
+from django.http import HttpResponse
+
+
+
+class SLA_request(TemplateView):
+    template_name = "make_queue/SLA.html"
+
+    def post(self, request):
+        self.contact_info = request.POST.get("email")
+        self.file = request.POST.get("stl_file")
+        self.content = request.POST.get("description")
+
+        msg = EmailMessage("SLA request", self.content + '\n' + self.contact_info, self.contact_info, ["test.makentnu@gmail.com"])
```

Comment:
The email address has to be changed from the testing one

Reply:
I think this might fail anyways, as it seems to try to send the message using the given email address. Which, if I'm not mistaken Google might not let us do.

Reply:
@zootos Would it work to use `noreply@makentnu.no` in the settings file, and override it with `sla-print@makentnu.no` here? As I understand it the email address used here was just for testing.

Anyway, the `reply-to` field should be set, as no one is receiving email on `sla-print@makentnu.no`.

Reply:
Yeah, see my new comment above :)